### ISSUE TITLE
Document Difference between Woodpecker and Drone

### ISSUE THREAD

**Issue by 6543** [woodpecker-ci/woodpecker#937]
For the biggest difference I would mention that,
Woodpecker is [Community](https://github.com/woodpecker-ci/woodpecker/blob/master/CONTRIBUTING.md) focused,
while last words on Drone Maintinance [Harness](https://harness.io/) does have.

on drone not all files are foss lisenced but are published under `Drone Enterprise License` ...
-> https://www.drone.io/enterprise/opensource/#features

tldr: for drone to scale you have to pay 
tldr: to contribute you have to sign a CLA -> the code is not yours anymore (they can re-license anytime they want)

**Comment by xoxys** [comment]
I'm still using Drone, but I would really like to switch to Woodpecker. What is blocking me is some missing features:

- No starlark/jsonnet support, which make complex pipelines quite hard to manage
- No org secrets
- No autoscaler

The missing autoscaler approach is the biggest issue for me. Personally, I don't need the CI every day, and I also don't need it 24/7 most of the time. Using the autoscaler is pretty cost-effective, as it can automatically spin up as much runner VMs as needed and destroy them afterwards, instead of running them all the time.

> tldr: for drone to scale you have to pay

That's not fully true, if you are a small company, non-profit org, open source developer, etc. (see [FAQ](https://docs.drone.io/enterprise/). Still, it's not fully open source, but at least the source code is publically available and can be reviewed. 

However, keep up the good work on Woodpecker :heart:

**Comment by 6543** [comment]
https://mastodon.technology/@WoodpeckerCI/107009887956204103
